R/config.R

```diff
@@ -304,7 +186,7 @@ write_config <- function(cfg, config_path, transitions_path, parameters_path,
   yaml::write_yaml(
     x = list(
       steps = ensure_int(cfg$steps),
-      max_delay = cfg$max_delay,
+      max_duration = cfg$max_duration,
```

Comment:
Note that this function (`write_config()`) as well as several others that operate on `config`s (e.g. `vary_param`, `graph_lifecycle`) are broken by these changes. My thinking is that we could tackle updating or removing these functions in a separate branch, since it's a separate question of what non-core model functionality we want to support in the package